crossing-training.lic

```diff
@@ -974,10 +1015,17 @@ class CrossingTraining
     dispose_trash(left_hand)
   end
 
+  def buy_wood
+    crafting_data = get_data('crafting')
+    return unless money_for_training?(700, 'Engineering')
+    order_item(crafting_data['shaping'][@settings.hometown]['stock-room'], 11)
+    bput('stow my lumber','You put your')
+  end
+  
   def buy_yarn
     return unless money_for_training?(700, 'Outfitting')
-    order_item(16_667, 13)
-    fput('stow my yarn')
+    order_item(crafting_data['tailoring'][@settings.hometown]['stock-room'], 13)
```

Comment:
I thought I had that in there.  Added the method to pull the data, similar to buy_wood